modules/meta_docs/help/meta_docs.md

```diff
@@ -0,0 +1,13 @@
+# Meta documentation module
```

Comment:
I don't think the source map file (not this line, the file above it..) is supposed to be included in this PR.

Reply:
@xlecours if you agree I can go ahead and remove it so we can get this in

Reply:
It could be useful to report bugs but those bugs would be in files that are not of our writing. We can probably remove the sourcemap indeed.